public/studio.h

```diff
@@ -1773,7 +1773,7 @@ struct thinModelVertices_t
 	{
 		Assert( ( m_numBoneInfluences  >= 1 ) && ( m_numBoneInfluences  <= 3 ) );
 		Assert( ( boneWeights.numbones >= 1 ) && ( boneWeights.numbones <= m_numBoneInfluences ) );
-		int    numStoredWeights = std::max( 0, ( m_numBoneInfluences - 1 ) );
+		int    numStoredWeights = max( 0, ( m_numBoneInfluences - 1 ) );
```

Comment:
What error is this supposed to fix?

Reply:
I do not remember the exact error but I'll go and find it again.

Reply:
hmm, thats odd, i cant seem to trigger the error again, ill just revert this commit unless it pops up again.

Reply:
![Screencap of the errors](https://github.com/danielga/sourcesdk-minimal/assets/17109660/4aab0136-84fd-4fd3-8867-c98c80aa6c5b)

Reply:
Can you revert these 2 lines then, please?

Reply:
Error popped up again as soon as I reverted the changes.